be/src/olap/rowset/segment_v2/column_writer.h

```diff
@@ -63,7 +63,7 @@ struct ColumnWriterOptions {
     bool need_inverted_index = false;
     uint8_t gram_size;
     uint16_t gram_bf_size;
-    std::vector<const TabletIndex*> indexes;
+    std::vector<const TabletIndex*> indexes; // unused
```

Comment:
Why not delete it?

Reply:
In the future, a column may support multiple indexes.